### DISCUSSION TITLE
Is there a non-case sensitive form of _eq?

### DISCUSSION THREAD

**Discussion by cosigyn** [directus/directus#11459] [Archive]
I have a search function for a specific field on my website hooked up to the API. Currently I'm having to use `?search` however that searches all fields, not just the one I am looking for.

`_eq` would be perfect however it is case sensitive so that is not a great option for me.

**Comment by azrikahar** [comment]
Currently this is not possible, but there is an existing discussion for this: https://github.com/directus/directus/discussions/2981. Although the title says "Search", but in this context the App UI search input is using filters like `_eq` which is what you are looking for.

It was sort of held back due to the fact that we'll have to ensure it works across different database vendors, but as the discussion there pointed out, the new changes over at Knex will help us gain more grounds on making this possible! With that said, let's consolidate this discussion over there moving forward 👍
